themes/MUG/invenio.cfg

```diff
@@ -280,3 +284,6 @@ OVERRIDE_RESOURCE_OVERVIEW = True
 
 THEME_FOOTER_TEMPLATE = "invenio_override/footer_mug.html"
 """footer template"""
+
+THEME_HEADER_LOGIN_TEMPLATE = "invenio_override/accounts/header_login_mug.html"
```

Comment:
```suggestion

```
the default template should work for all usecases.

Reply:
Thank you for pointing that out. I’ll fix the issue in our override and update the PR's.